### ISSUE TITLE
[Bug/Crash] Error: Cannot read properties of undefined (reading 'path')

### ISSUE THREAD

**Issue by LunarN0v4** [digimezzo/dopamine#718]
Relevant logs:
```
[2024-10-23 20:17:02.301] [info]  [ScrobblingService] [handlePlaybackStartedAsync] Successfully updated Now Playing for track 'K-391 - Die Alone'
[2024-10-23 20:17:04.673] [info]  [TrackService] [getTracksForAlbums] Finished getting tracks for albums. Time required: 3 ms
[2024-10-23 20:17:04.674] [info]  [TrackSorter] [sortByAlbum] Finished sorting tracks by album. Time required: 0 ms
[2024-10-23 20:17:04.674] [info]  [TrackBrowserComponent] [orderTracks] Finished ordering tracks. Time required: 0 ms
[2024-10-23 20:17:04.837] [error] [GlobalErrorHandler] [handleError]  Handling global error. Error: Cannot read properties of undefined (reading 'path')
[2024-10-23 20:17:04.838] [info]  [GlobalErrorHandler] [showGlobalErrorDialog] Showing global error dialog
[2024-10-23 20:17:06.428] [info]  [GlobalErrorHandler] [showGlobalErrorDialog] Closing application
[2024-10-23 20:17:06.436] [info]  [QueuePersister] [save] Saving queue
[2024-10-23 20:17:06.459] [info]  [QueuePersister] [save] Saved queue of 1 tracks
```
OS Information:
```
OS: GNU/Linux
Kernel: Linux 6.11.4-arch2-1
Distribution: Arch
```

**Comment by lokopeto** [comment]
i am having the same problem with this new update, in addition to that, now tracks has a delay to play

**Comment by digimezzo** [comment]
@LunarN0v4, @lokopeto I'm fiing that problem currently.

@lokopeto Coudl you tell me more about that delay? I don't have that here, so I'll need all info you can get me in order to fix it.

**Comment by lokopeto** [comment]
the delay occurs when you skip musics fast, for some reason, i can't replicate anymore, the problem just disappeared i think, it was more often when i updated it, i have 6.000 musics and i use the folders page a lot.

i gonna use this build, if i see something i will tell you, thanks

**Comment by bryanparadis** [comment]
I have this issue as well on Windows. My application is stuck like this and I have to delete my data to fix it.

[Dopamine.log](https://github.com/user-attachments/files/17891665/Dopamine.log)

**Comment by digimezzo** [comment]
Fixed in 9a53f80
